tsk/fs/tsk_fs.h

```diff
@@ -676,6 +676,7 @@ extern "C" {
         void *a_ptr);
     extern size_t tsk_fs_dir_getsize(const TSK_FS_DIR *);
     extern TSK_FS_FILE *tsk_fs_dir_get(const TSK_FS_DIR *, size_t);
+    extern TSK_FS_FILE *tsk_fs_dir_get2(const TSK_FS_DIR *, size_t, size_t load_attributes);
```

Comment:
This is going into the public API. I don't think it needs a "2" in the name.

Reply:
`tsk_fs_dir_get()` is the legacy API. We can't change it. 
`tsk_fs_dir_get2()` is the new API that takes 2 arguments. What would you rather call it?